_apidocs/get-opportunities-public-api.md

```diff
@@ -89,6 +89,7 @@ setAside	|Set Aside Description	|String | v1<br> v2
 setAsideCode	|Set Aside Code	|String | v1<br> v2
 reponseDeadLine	|Response Deadline Date	|String | v1<br> v2
 naicsCode	|NAICS Code. This code is maximum of 6 digits	|String | v1<br> v2
+naicsCodes	|NAICS Codes. This array contains multiple codes.This code is maximum of 6 digits  |String[] | v1<br> v2
```

Comment:
There are multiple examples and you also need to update those response example as well.